app/models/person.rb

```diff
@@ -21,10 +21,14 @@ class Person < ActiveRecord::Base
   has_many :called_legislators, through: :calls
 
   VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-]+(\.[a-z\d\-]+)*\.[a-z]+\z/i
-  validates :email, presence: true, uniqueness: { case_sensitive: false },
-                    format: { with: VALID_EMAIL_REGEX }
+  validates :email, uniqueness: { case_sensitive: false },
+                    format: { with: VALID_EMAIL_REGEX },
+                    allow_nil: true
 
-  before_save { self.email = email.downcase }
+  validates :email, presence: true, if: "phone.nil?"
+  validates :phone, presence: true, if: "email.nil?"
```

Comment:
I personally like the way `validates :email, presence: true, unless: :phone` looks, but that may just be because symbols appear in blue in my text editor :)